services/docker-compose.yml

```diff
@@ -1132,15 +1141,17 @@ services:
         "--databases",
         "8",
         "--appendonly",
-        "yes"
+        "yes",
+        "--requirepass",
+        "${REDIS_PASSWORD}"
       ]
     networks:
       - default
       - autoscaling_subnet
     volumes:
       - redis-data:/data
     healthcheck:
-      test: [ "CMD", "redis-cli", "ping" ]
+      test: [ "CMD", "redis-cli", "-a", "${REDIS_PASSWORD}", "ping" ]
```

Comment:
no long named option instead of -a ?

Reply:
I double-checked an it seems that there is no second, long-name-style option subsituting "-a"